user-impl/src/main/java/Module.java

```diff
@@ -1,12 +1,13 @@
-package com.example.auction.user.impl;
-
 import com.example.auction.user.api.UserService;
+import com.example.auction.user.impl.UserRepository;
+import com.example.auction.user.impl.UserServiceImpl;
 import com.google.inject.AbstractModule;
 import com.lightbend.lagom.javadsl.server.ServiceGuiceSupport;
 
-public class UserModule extends AbstractModule implements ServiceGuiceSupport {
+public class Module extends AbstractModule implements ServiceGuiceSupport {
```

Comment:
Wow, that is some very good detective work! How strange.

We have noticed some other issues related to this in the past, for example, https://github.com/lagom/lagom/issues/568. That issue pertains to JDBC rather than Cassandra, but I think it's similar in that the underlying issue is a lack of coordination in the initialization order. All of our `ClusterStartupTask` implementations kick off immediately as soon as they are defined.

@rcavalcanti: could you record your findings in a lagom/lagom issue? We should really fix this upstream.